clients/vnetClient/vnetClient.go

```diff
@@ -0,0 +1,52 @@
+package vnetClient
+
+import (
+	"encoding/xml"
+	"fmt"
+	azure "github.com/MSOpenTech/azure-sdk-for-go"
+)
+
+const (
+	azureNetworkConfigurationURL = "services/networking/media"
+)
+
+//GetVirtualNetworkConfiguration retreives the current virtual network
+//configuration for the currently active subscription. Note that the
+//underlying Azure API means that network related operations are not safe
+//for running concurrently.
+func GetVirtualNetworkConfiguration() (*NetworkConfiguration, error) {
+	response, err := azure.SendAzureGetRequest(azureNetworkConfigurationURL)
+	if err != nil {
+		return nil, err
+	}
+
+	networkConfiguration := &NetworkConfiguration{}
+	err = xml.Unmarshal(response, &networkConfiguration)
+	if err != nil {
+		return nil, err
+	}
+
+	return networkConfiguration, nil
+}
+
+//SetVirtualNetworkConfiguration configures the virtual networks for the
+//currently active subscription according to the NetworkConfiguration given.
+//Note that the underlying Azure API means that network related operations
+//are not safe for running concurrently.
+func SetVirtualNetworkConfiguration(networkConfiguration *NetworkConfiguration) error {
+	networkConfiguration.setXmlNamespaces()
+	networkConfigurationBytes, err := xml.Marshal(networkConfiguration)
+	if err != nil {
+		return err
+	}
+
+	fmt.Println("Content", string(networkConfigurationBytes))
```

Comment:
Nope, that was an oversight from when I was trying to debug the schema. I'll remove.